css/css-backgrounds/box-shadow-overlapping-002.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Background and Borders Test: box-shadow and overlapping of adjacent text</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#shadow-layers" >
+  <link rel="match" href="reference/box-shadow-overlapping-001-ref.html">
+  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+
+  <meta content="" name="flags">
+  <meta content="Box shadows do not affect layout. Box shadows take up no space. Box shadows do not affect or influence normal flow of boxes. Therefore, box shadows can 'collide' with other boxes and can overlap other boxes (inline box or line box) or be overlapped by other boxes. In this test, the text ('PNG') follows a right outer box shadow and therefore overlaps such shadow. The box shadow, even though it is overlapped by the text ('PNG'), nevertheless overlaps the background color (yellow) of the block element." name="assert">
+
+  <style>
+  div
+    {
+      background-color: yellow;
+      color: green;
+      float: left;
+      font-family: Ahem;
+      font-size: 100px;
+      line-height: 1;
```

Comment:
font: 100px/1 Ahem;

There's no reason to be wordy here.

Reply:
There is a reason why. If I need to see where the P, E, D, P, N, G glyphs are, then I can toggle on/off the font-family in the web browser inspection tool. I would not be able to do this easily and quickly if I had used a shorthand form. Use of longhand form for font related properties is only for convenience.